src/coreclr/vm/ecalllist.h

```diff
@@ -768,7 +764,6 @@ FCClassElement("ArgIterator", "System", gVarArgFuncs)
 FCClassElement("Array", "System", gArrayFuncs)
 FCClassElement("AssemblyBuilder", "System.Reflection.Emit", gAssemblyBuilderFuncs)
 FCClassElement("AssemblyLoadContext", "System.Runtime.Loader", gAssemblyLoadContextFuncs)
-FCClassElement("AssemblyName", "System.Reflection", gAssemblyNameFuncs)
```

Comment:
nit - these three are also dead/unused:

```c++
FCClassElement("IEnumerable", "System.Collections", gStdMngIEnumerableFuncs)
FCClassElement("IEnumerator", "System.Collections", gStdMngIEnumeratorFuncs)
FCClassElement("IReflect", "System.Reflection", gStdMngIReflectFuncs)
```

Reply:
They come from `#include "mngstditflist.h"`

Reply:
Ah string concatenation. I was searching for the whole word (`gStdMngIEnumerableFuncs`).

Reply:
I also could not find these and removed them. But everything is passing. Do we have tests for this?

Reply:
This is very specialized COM interop that we tried to avoid touching. It is possible that the tests are in the outer runs only, or does not exist at all.

Reply:
Some interop tests started failing eventually. I will put these back. 
It could be nicer if removal caused some kind of build break